.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Build docs
+
+jobs:
+  build:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements/build.txt
+          python -m pip install -r requirements/default.txt
+          python -m pip install -r requirements/optional.txt
+          python -m pip install -r requirements/docs.txt
+      - name: Install package
+        run: |
+          python setup.py install
```

Comment:
Could we use `python -m pip install .` here? That will future-proof us for when we switch to pyproject.toml etc.

Reply:
ok! done.